frontend/src/i18n/locales/de.yaml

```
@@ -113,6 +113,12 @@ video:
   more-from-series: Mehr von „{{series}}“
   deleted-video-block: Das hier referenzierte Video wurde gelöscht.
   not-allowed-video-block: Sie sind nicht autorisiert, das hier eingebettete Video zu sehen.
+  not-ready:
+    title: Video noch nicht verarbeitet
+    text: >
+      Das gewünschte Video ist leider noch nicht von Opencast verarbeitet worden. Dies sollte
```

```suggestion
      Das gewünschte Video ist leider noch nicht verarbeitet worden. Dies sollte
```

Maybe? Again, I think most users have no idea what "Opencast" is. And it's really not important here, right? (The same in English, of course)

---

:shrug: sure